### PR TITLE
Added query string handling with tests

### PR DESCRIPTION
**Pull request by croz1007**

### CODE REVIEW

File: lib/cbus_elixir_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -3,10 +3,20 @@ defmodule CbusElixirWeb.PageController do
...
 
-  def index(conn, _params) do
-    next_meeting = Meetings.next_meeting(Timex.today())
-
+  def index(conn, params) do
+    next_meeting = Meetings.next_meeting(parse_meeting_date(params["date"]))
     render conn, "index.html", next_meeting: next_meeting
   end
 
+  defp parse_meeting_date(nil), do: Timex.today()
```
Comment (mdarby) [inline lib/cbus_elixir_web/controllers/page_controller.ex]: yusssssss guard clauses FTW

File: lib/cbus_elixir_web/controllers/page_controller.ex [hunk 2]
```diff
@@ -3,10 +3,20 @@ defmodule CbusElixirWeb.PageController do
...
+
+  defp parse_meeting_date(date) do
+    date
+    |> Date.from_iso8601() 
+    |> case do
+      {:ok, result} -> result
+      {:error, _ } -> Timex.today()
```
Comment (mdarby) [inline lib/cbus_elixir_web/controllers/page_controller.ex]: So clean.

File: test/cbus_elixir_web/controllers/page_controller_test.exs [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+    date = "2018-03-25"
+    conn = get conn, "/?date=#{date}"
+    assert html_response(conn, 200) =~ "2018-04-03"
+  end
+
+  test "Invalid Date Query String", %{conn: conn} do
+    date = "20180325"
+    conn = get conn, "/?date=#{date}"
+    assert html_response(conn, 200) =~ "2018-03-06"
+  end
```
Comment (mdarby) [inline test/cbus_elixir_web/controllers/page_controller_test.exs]: YES

**Review by mdarby — DISMISSED** [review]
Fucking aye

**Review by Kartstig — APPROVED** [review]
LGTM

**Pull request merged**